akka-bootstrapper/src/main/scala/filodb/akkabootstrapper/ClusterSeedDiscovery.scala

```diff
@@ -24,40 +24,44 @@ abstract class ClusterSeedDiscovery(val cluster: Cluster,
   }
 
   @throws(classOf[DiscoveryTimeoutException])
+  //scalastyle:off null
   protected def discoverPeersForNewCluster: Seq[Address]
 
   protected def discoverExistingCluster: Seq[Address] = {
 
     val seedsEndpoint = settings.seedsBaseUrl + settings.seedsPath
+    var response: HttpResponse[String] = null
+    var retriesRemaining = settings.seedsHttpRetries
+    do {
+      try {
+        logger.info(s"Trying to fetch seeds from $seedsEndpoint ... $retriesRemaining retries remaining.")
+        response = Http(seedsEndpoint).timeout(2000, 2000).asString
```

Comment:
should this timeout be configurable also?

Reply:
Lets leave that out for now.